src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -44,10 +46,11 @@
   private static final String HINT3_PARAMETER = "hint3";
   private static final String OBSCURITY_PARAMETER = "obscurity";
   private static final String TAG_PARAMETER = "tag";
-  private static final String REDIRECT_URL = "/destination-data";
+  private static final String REDIRECT_URL = "/index.html";
 
-  // Temporarily stores the destination created by the user
-  public List<Destination> destinations = new ArrayList<>();
+  private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
+
+  private static final Gson GSON = new Gson();
```

Comment:
Same here, this probably shouldn't be capitalized.

Reply:
On the other hand, I think that this is a constant according to the definition above. AFAICT, none of `Gson`'s methods have detectable side effects and its observable state cannot change. I think this name should stay as-is.

Reply:
That's fine.  It seemed incorrect to me but I agree with you now.